routes/delete-project.js

```diff
@@ -27,6 +27,10 @@ module.exports = function(config) {
         return;
       }
 
+      if(req.session.project && req.session.project.meta.id == projectId) {
```

Comment:
I don't think so! We have control over that code, so if the `project` exists (which it may or may not depending on whether or not the user has never opened a project in thimble) we know that it will contain all the necessary properties!